src/Bigfile.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+// inspirated by:https://dzone.com/articles/processing-large-files-using-php
+
+namespace Spatie\DbSnapshots;
+
+class Bigfile
+{
+    protected $file;
+
+    public function __construct($filename, $mode = 'r', $compressed = false)
+    {
+        if (! file_exists($filename)) {
+            throw new \Exception('File not found');
+        }
+        if ($compressed) {
```

Comment:
Use a ternary so we can lose that `else` statement.

Reply:
changed